mono/metadata/custom-attrs.c

```diff
@@ -1951,8 +1951,6 @@ mono_reflection_get_custom_attrs_by_type_handle (MonoObjectHandle obj, MonoClass
 		MONO_HANDLE_ASSIGN (result, MONO_HANDLE_NEW (MonoArray, mono_custom_attrs_construct_by_type (cinfo, attr_klass, error))); /* FIXME use coop handles for mono_custom_attrs_construct_by_type */
 		if (!cinfo->cached)
 			mono_custom_attrs_free (cinfo);
-		if (!result)
```

Comment:
Don't we still need this null check?

Reply:
It should land at the same place w/ or w/o the check.

Reply:
Should have been `if (MONO_HANDLE_IS_NULL (result)) goto leave`, I guess?  Or better `goto_if_nok (error, leave)`.